custom/icds_reports/static/js/directives/prevalence-of-stunting-report/prevalence-of-stunting-report.directive.js

```diff
@@ -4,11 +4,14 @@ var url = hqImport('hqwebapp/js/initial_page_data').reverse;
 
 function PrevalenceOfStuntingReportController($scope, $routeParams, $location, $filter, maternalChildService,
     locationsService, dateHelperService, navigationService, userLocationId, storageService,  genders, ages, haveAccessToAllLocations,
-    baseControllersService, haveAccessToFeatures, isAlertActive) {
+    baseControllersService, haveAccessToFeatures, isAlertActive, isMobile) {
```

Comment:
Worth reading this commit message, since it provides a template for a series of commits that come next (although one of the reports gets broken out for no good reason). I'll identify the elements of the commit message by number.

This change is part of number 3.